src/org-manager/services/pba.service.ts

```diff
@@ -17,7 +18,7 @@ export class PbaService {
   }
 
   public setPBAStatus(body: any): Observable<any> {
-    return this.http.put<any>(`${this.pbaStatusUrl}`, body);
+     return this.http.put<ConfirmNewPBAModel>(`${this.pbaStatusUrl}`, body);
```

Comment:
Great, just need that return type on the function as well

Reply:
I've changed the return type as well now. Thanks @paddybasi .